authentication/templates/register.html

```diff
@@ -0,0 +1,63 @@
+<html>
+<head>
+  
+  {% block title %}
+  <title>Library</title>
+  {% endblock %}
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.bundle.min.js"></script>
+  {% load static %}
+</head>
+
+<body>
+
+    <div class="container-fluid">
+
+        <div class="row">
+        <div class="col-sm-2 mt-5">
+        {% block sidebar %}
+        <ul class="sidebar-nav">
+            <li><a href="{% url 'index' %}">Home</a></li>
+            <li><a href="{% url 'book-list' %}">All books</a></li>
+        </ul>
+        
+        <ul class="sidebar-nav">
+          {% if user.is_authenticated %}
+            <li>User: {{ user.username }}</li>
+            <li><a href="{% url 'view-loaned' %}">My Borrowed</a></li>
+            <li><a href="{% url 'logoutView' %}">Logout</a></li>   
+          {% else %}
+            <li><a href="{% url 'loginView' %}">Login</a></li>
+          {% endif %} 
+        </ul>
```

Comment:
Always prefer to use base.html to make the code DRY.

Reply:
I always use base.html for here for only two HTML files I add upper base.html